include/Main.hpp

```diff
@@ -83,6 +85,24 @@ class AgISOVirtualTerminalApplication : public juce::JUCEApplication
 		// the other instance's command-line arguments were.
 	}
 
+	static void onCrash(void *)
+	{
+		auto stackTrace = SystemStats::getStackBacktrace();
+		auto appDataDir = File(File::getSpecialLocation(File::userApplicationDataDirectory).getFullPathName() +
+		                       File::getSeparatorString() +
+		                       "Open-Agriculture" +
+		                       File::getSeparatorString());
+		String timestamp(static_cast<uint64_t>(std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count()));
```

Comment:
Would not be better to add "human" datetime to the filename? CAN logs have the same format, it would be easier to match them with eyes I think.

Reply:
Sure, I can use the same timestamp format as the logs. I will update it